dev_guide/deployments/deployment_strategies.adoc

```diff
@@ -52,28 +51,28 @@ downtime may be needed to complete the transition.
 - If you have a non-isolated test environment, you can break both new and old versions.
 
 
-Since the end user usually accesses the application through a route handled by a 
-router. The deployment strategy can focus on deployment configuration features or 
-routing features.
+Since the end user usually accesses the application through a route handled by a
```

Comment:
This is an incomplete sentence. Combine the two?

router, the deployment strategy can focus